README.md

```diff
@@ -46,7 +41,55 @@ To setup your project using a local python virtual environment, follow these ins
     cd {{ project_name }}
     ```
 
-2. Setup the Python Dependencies
+2. Create the .env file
+
+    An `.env` file is requird to run the application. It can be created from the `.env.sample` either manually or with the create-envfile.py script.
+
+    The script accepts several parameters to create the file, in detail:
+
+    **required**:
+  
+    - hostname: e.g. master.demo.geonode.org
+
+    **optional:**
+
+    - *https*: (boolean), default value is False
+    - *email*: Admin email (this is required if https is set to True since a valid email is required by Letsencrypt certbot)
+    - *emv_type*: `prod`, `test` or `dev`. It will set the `DEBUG` variable to `False` (`prod`, `test`) or `True` (`dev`)
+    - *geonodepwd*: GeoNode admin password (required inside the .env)
+    - *geoserverpwd*: Geoserver admin password (required inside the .env)
+    - *pgpwd*: PostgreSQL password (required inside the .env)
+    - *dbpwd*: GeoNode DB user password (required inside the .env)
+    - *geodbpwd*: Geodatabase user password (required inside the .env)
+    - *clientid*: Oauth2 client id (required inside the .env)
+    - *clientsecret*: Oauth2 client secret (required inside the .env)
+    - *secret key*: Django secret key (required inside the .env)
+    - *sample_file*: absolute path to a env_sample file used to create the env_file. If not provided, the one inside the GeoNode project is used.
+    - *file*: absolute path to a json file that contains all the above configuration
+
+     **NOTE:**
+    - if the same configuration is passed in the json file and as an argument, the CLI one will overwrite the one in the JSON file
```

Comment:
Can we put an example of `file.json` contents please?

Reply:
I'll add it to the README, meanwhile if you want to see it, is present also in the issue